chrobalt/build/configs/variables.gni

```diff
@@ -0,0 +1,47 @@
+# Copyright 2024 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Get the path to the starboard implementation and include its GN
+# configuration.
+
+
+import("//$starboard_path/platform_configuration/configuration.gni")
+
+sb_is_modular = sb_is_evergreen || build_with_separate_cobalt_toolchain
+
+declare_args() {
+  use_tsan = getenv("USE_TSAN") == 1
+}
+
+_use_asan_default =
+    (is_debug || is_devel) && (is_linux || is_mac) && is_clang &&
```

Comment:
nit: Do we support Chrobalt on Mac? (since |is_mac| is used here).

Reply:
I have copied this over from starboard/../BUILDCONFIG

https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:starboard/build/config/BUILDCONFIG.gn;l=130?q=repo:lbshell-internal%2Fcobalt_src%20AND%20branch:COBALT%20BUILDCONFIG

At some point when we have Chrobalt for mac , this will be enabled by default which i think should be the expected behaviour